src/root/rootReducer.ts

```diff
@@ -39,6 +40,7 @@ const createRootReducer = () =>
     genome,
     global,
     speciesSelector,
+    newSpeciesSelector, // TODO: rename to just species slector when we retire the old one
```

Comment:
Shouldn't this be going into the server side reducer as well?

Reply:
We'll investigate this in ENSWBSITES-2019